easybuild/easyconfigs/p/PHANOTATE/PHANOTATE-20190724-cc42101-foss-2018b.eb

```diff
@@ -0,0 +1,45 @@
+# This file is an EasyBuild reciPY as per https://easybuilders.github.io/easybuild/
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics
+
+easyblock = 'ConfigureMake'
+
+name = 'PHANOTATE'
+git_commit = 'cc42101'
+version = '20190724-%s' % (git_commit)
+
+homepage = 'https://github.com/deprekate/PHANOTATE'
+description = "PHANOTATE: a tool to annotate phage genomes"
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+sources = [{
+    'filename': 'PHANOTATE-%s.tar.gz' % version,
+    'git_config': {
+        'url': 'https://github.com/deprekate',
+        'repo_name': 'PHANOTATE',
+        'commit': '%s' % git_commit,
+        'recursive': True,
```

Comment:
Hmm, the installation of this is broken, because the current `master` of the `PHANOTATE` repo doesn't have submodules anymore.

So, the initial clone with `git clone --recursive` doesn't create a `fastpath` directory, and the subsequent `git checkout cc42101` just leaves an empty `fastpath` directory.

The subsequent `git submodule update` doesn't do much, it seems like that should be `git submodule update --init` to ensure the submodule gets initialized first (so that needs to be fixed in framework).